### DISCUSSION TITLE
Question re. tab arrows

### DISCUSSION THREAD

**Discussion by mantacid** [tsujan/Kvantum#890] [Q&A]
I had --somehow-- accidentally deleted my custom kvantum theme, but remade it from another one I had made. however, when I checked the preview, I discovered that the tab arrows looked like they do in the image. Why do they look like this, and how do I fix them.
![Screenshot_20230704_162212](https://github.com/tsujan/Kvantum/assets/123284210/b0fbb822-4424-4ce1-9c12-785128f15e54)

**Comment by tsujan** [comment]
Tab-bar scroll buttons are painted as small tool-buttons with arrows but, for technical reasons, their background rectangle is filled with the window color. Depending on the theme, this filling may be more or less visible inside some widgets.
